backend/corpora/dataset_processing/process.py

```diff
@@ -691,16 +668,19 @@ def main():
             logger.error(f"Step function configuration error: Unexpected STEP_NAME '{step_name}'")
 
     except ProcessingCancelled:
-        cancel_dataset(dataset_id)
+        # cancel_dataset(dataset_id)
```

Comment:
Is the cancellation deprecated? If not, please leave a TODO since we'll have to keep this feature.

Reply:
I'm thinking that it should be deprecated. Tt looks like all it does is just delete the assets, and I thought we wanted to avoid deleting assets to allow going back in history.